ftplugin/latex-suite/compiler.vim

```diff
@@ -591,7 +591,7 @@ function! Tex_CompileMultipleTimes()
 		" The first time we see if we need to generate the bibliography and if the .bbl file
 		" changes, we will rerun latex.
 		" We use '\\bibdata' as a check for BibTeX and '\\abx' as a check for biber.
-		if runCount == 0 && Tex_IsPresentInFile('\v\\bibdata|\\abx', mainFileName_root.'.aux')
+		if runCount == 0 && Tex_IsPresentInFile('\\bibdata|\\abx', mainFileName_root.'.aux')
```

Comment:
When I am editing the file paper.tex in vim. The output is
```
:echo Tex_GetMainFileName(':p:t:r')
paper
```


I attached the paper.aux file which xelatex generated. There is a \bibdata in the file, but the regular expression cannot match with `\v`.
For example, when I type the following commands in vim, the outputs are different:
```
:echo Tex_IsPresentInFile('\\bibdata|\\abx', 'paper.aux')
1
:echo Tex_IsPresentInFile('\v\\bibdata|\\abx', 'paper.aux')
0
```

[paper.aux.txt](https://github.com/vim-latex/vim-latex/files/216478/paper.aux.txt)